plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveConnectorTest.java

```diff
@@ -3149,8 +3149,7 @@ public void testPartitionPerScanLimitWithMultiplePartitionColumns()
 
```

Comment:
Can we add a test that properites are derived even when predicates were not pushed down (if there wasn't such a test)?
e.g join on two partitioned tables should contain predicates derived from properties